network/message/authorization.go

```diff
@@ -34,18 +35,18 @@ type MsgAuthConfig struct {
 // Expected error returns during normal operations:
 //   - ErrUnauthorizedMessageOnChannel: the channel is not included in the message's list of authorized channels
```

Comment:
please add the new broadcast error here too

Reply:
https://github.com/onflow/flow-go/pull/3525/commits/44f438d01acf94316e2f4b8cf9466a51a7301ead